apps/supernova/src/workers/alerts.js

```diff
@@ -3,20 +3,25 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import ApiService from "../api/apiService"
-import { get } from "../api/client"
 import { sortAlerts, countAlerts } from "../lib/utils"
-
 let compareAlertString
 
-/**
- * @param {string} endpoint
- * @param {object} options
- * @returns {function} fetch function
- */
-const fetchAction = (endpoint, options = {}) => {
-  return get(`${endpoint}/alerts`, { params: options.params }).then((items) => {
-    // sort alerts
+export const fetchAlerts = async (endpoint) => {
```

Comment:
I would recommed to setup react Query with all queries and functions in a central place  based on the idea in heureka:
https://github.com/cloudoperators/juno/blob/b007ad29c901861f480078f5b6f64e933c482a19/apps/heureka/src/hooks/useQueryClientFn.js#L20